lib/trento_web/openapi/v1/schema/activity_log.ex

```diff
@@ -37,7 +35,26 @@ defmodule TrentoWeb.OpenApi.V1.Schema.ActivityLog do
         },
         required: [:id, :type, :actor, :metadata, :occurred_on]
       }
-    },
-    struct?: false
-  )
+    })
+  end
+
+  defmodule ActivityLog do
+    @moduledoc false
+
+    OpenApiSpex.schema(
+      %{
+        title: "ActivityLog",
+        description: "Activity Log for the current installation.",
+        type: :object,
+        properties: %{
+          data: ActivityLogEntry,
```

Comment:
`ActivityLogEntry` should either be `ActivityLogEntries` or just the whole `ActivityLog`